src/pages/ProgramContentPage/ProgramContentExerciseBlock.tsx

```diff
@@ -62,10 +64,11 @@ const ProgramContentExerciseBlock: React.VFC<{
   )
 }
 
-const useLastExercise = (programContentId: string, exerciseId?: string | null) => {
+const useLastExercise = (programContentId: string, memberId: string | null, exerciseId?: string | null) => {
   const condition: hasura.GET_LAST_EXERCISEVariables['condition'] = {
```

Comment:
Why is the type of memberId `string | null` ? Would it be the same as `exerciseId` ?
ex: 
```
const useLastExercise = (programContentId: string, memberId?: string, exerciseId?: string | null) => {
    id: exerciseId ? { _eq: exerciseId } : undefined,
    program_content_id: { _eq: programContentId },
    member_id: memberId ? { _eq: memberId }: undefined,
```

or the memberId is the required input in useLastExercise hook, it can not be null.

Reply:
I think all of the variables is required